CMSgov/easi-app
# [EASI-3268] Delete prince references on backend

# EASI-3268

## Changes and Description

- Delete references (configs, env vars, handlers, services) from backend code now that #2198 is complete

## How to test this change

- Make sure the app compiles and tests pass!
- Make sure PDF rendering still works on the frontend (really should, given that the code deleted should be completel unused)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
